examples/diff_drive_odometry.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2021 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import math
+
+from ignition.math import Angle, DiffDriveOdometry
+
+odom = DiffDriveOdometry()
+
+wheelSeparation = 2.0
+wheelRadius = 0.5
+wheelCircumference = 2 * 3.1416 * wheelRadius
+
+# This is the linear distance traveled per degree of wheel rotation.
+distPerDegree = wheelCircumference / 360.0
+
+# Setup the wheel parameters, and initialize
+odom.set_wheel_params(wheelSeparation, wheelRadius, wheelRadius)
+startTime = datetime.datetime.now()
+odom.init(datetime.timedelta())
+
+print('--- Rotate the both wheels by 1 degree. ---')
+time1 = startTime + datetime.timedelta(milliseconds=100)
+odom.update(Angle(1.0 * 3.1416 / 180),
+            Angle(1.0 * 3.1416 / 180),
+            time1 - startTime)
+
+print('Linear velocity: {} Odom linear velocity: {}'.
+    format(distPerDegree / 0.1, odom.linear_velocity()))
+
+print('Angular velocity should be zero since the "robot" is traveling' +
+      ' in a straight line:\n' +
+      '\tOdom angular velocity: {}'
+      .format(odom.angular_velocity()))
+
+# Sleep again, this time rotate the right wheel by 1 degree.
+print('--- This time rotate the right wheel by 1 degree. ---');
```

Comment:
Isn't the left wheel being rotated by 1 degree (2 - 1), and the right wheel by 2 (3 - 1)?